resources/lib/comm.py

```diff
@@ -0,0 +1,143 @@
+import classes
+import config
+import json
+
+from aussieaddonscommon import session
+
+
+def fetch_url(url, headers={}):
+    """
+    Use custom session to grab URL and return the text
+    """
+    with session.Session(force_tlsv1=True) as sess:
+        res = sess.get(url, headers=headers)
+        return res.text
+
+
+def list_series():
+    """
+    Create and return list of series objects
+    """
+    res = fetch_url(config.TVSERIES_URL)
+    data = json.loads(res)
+    listing = []
+    for show in data['items']:
+        for season in reversed(show['containsSeason']):
```

Comment:
Just wonder if you want some sort of try/except here in case it fails on one show for example?

Reply:
The function that calls it in menu.py is in a try/except for the issue reporting - is that what you mean?

Reply:
Just thinking that if you have a case where one particular show can't be parsed, it'll throw an exception and nothing will work. If we add a try/except around the parsing for each show, we could skip a bad one, but still show the list with all the others. 

If there was an issue, this would mask it (so harder for us to find it) but would be a better experience for the user. I wouldn't -1 you for not doing it though.. just something I've been thinking about in my other addons.

Reply:
I was thinking the same thing, that we'd be missing the error reports. Now that we've got travis making releases so much simpler/quicker it's easier to push quick fixes if something does pop up.

how about a 
```python
if not show.get('containsSeason'):
    continue
```
?

Reply:
Yep, or make it less negative with 
```
if show.get('constainsSeason'):
  do things
```
Would that also work?

Reply:
yep, can do. I'll try and make the rest of the function safer by using the get method to access the items rather than just by name.